src/main.rs

```diff
@@ -18,12 +19,19 @@ async fn main() -> Result<()> {
     }
     match opts.subcmd {
         None => {
-            let mut force: HashSet<String> = HashSet::new();
-            for f in opts.force {
-                force.insert(f);
-            }
+            let force: HashSet<String> = HashSet::from_iter(opts.force);
+            let force_all = opts.force_all;
+            let is_force = |path| {
```

Comment:
I think rewrite this to `force_all || force.contains(&path)` is less readable?

Reply:
what happen if force and force_all appear in the same time

Reply:
> what happen if force and force_all appear in the same time

Just use force_all and force will be ignored.

Reply:
> I think rewrite this to `force_all || force.contains(&path)` is less readable?

I think so

Reply:
> force_all || force.contains(&path)

I think `force_all || force.contains(&path)` is more readable

Reply:
Updated.